src/rabbit_auth_backend_internal.erl

```diff
@@ -321,6 +334,23 @@ list_permissions(Keys, QueryThunk) ->
             %% TODO: use dirty ops instead
             rabbit_misc:execute_mnesia_transaction(QueryThunk)].
 
+list_permissions(Keys, QueryThunk, Ref, AggregatorPid) ->
+    [AggregatorPid ! {Ref, filter_props(Keys, [{user,      Username},
```

Comment:
Before I was doing a code review just looking at the diff here on Github. Inspecting the full code tells me all the 
"async" functions just duplicated code from the module. Is my assessment correct?
https://github.com/rabbitmq/rabbitmq-server/blob/rabbitmq-server-62/src/rabbit_auth_backend_internal.erl#L324